JustEat.StatsD.Common.props

```diff
@@ -0,0 +1,21 @@
+﻿<Project>
+  <PropertyGroup>
+    <Authors>Just Eat</Authors>
+    <Company>Just Eat</Company>
+    <Copyright>Copyright (c) Just Eat 2015-2017</Copyright>
```

Comment:
Just realised could go one step further with this and use MSBuild to calculate the end year for the copyright...